hedy_content.py

```diff
@@ -543,7 +543,7 @@ def get_sorted_adventure_programs(self, programs, adventure_names):
         programs_by_adventure = []
         for item in programs:
             programs_by_adventure.append(
-                {'adventure_name': adventure_names.get(item.get('adventure_name')),
+                {'adventure_name': adventure_names.get(item.get('adventure_name'), item.get('adventure_name')),
```

Comment:
What does this new line do?

Reply:
if you give a second argument to a get, it will return that second argument when nothing is found (see: https://365datascience.com/question/what-does-the-second-argument-of-the-get-method-stand-for/)

Reply:
Sweet good to know!